site/app/templates/grading/simple/Display.twig

```diff
@@ -72,6 +72,32 @@
         </div>
     </div>
 
+    <p>Sort By:</p>
+    <div style="margin: 1em 0;"> {# This is a drop down menu containing sorting options #}
+        <select id="subsection_sort" name="subsection_sorting" onChange="window.location.href=this.value" style="font-size:15px;">
+            <option value="{{ grading_url }}?view={{ view_all ? 'all' : '' }}&sort=section_subsection" aria-label="sort by Section then Subsection">Section then Subsection then User ID</option>
+            <option value="{{ grading_url }}?view={{ view_all ? 'all' : '' }}&sort=id" aria-label="sort by ID" >Section then User ID</option>
+            <option value="{{ grading_url }}?view={{ view_all ? 'all' : '' }}&sort=first" aria-label="sort by First Name">Section then First Name then User ID</option>
+            <option value="{{ grading_url }}?view={{ view_all ? 'all' : '' }}&sort=last" aria-label="sort by Last Name">Section then Last Name then User ID</option>
+        </select>
+        {% if sort == "section_subsection" %}
+            <script>
+                $('select').val( "{{ grading_url }}?view={{ view_all ? 'all' : '' }}&sort=section_subsection" )
+            </script>
+        {% elseif sort == "id"  %}
+            <script>
+                $('select').val( "{{ grading_url }}?view={{ view_all ? 'all' : '' }}&sort=id" )
+            </script>
+        {% elseif sort == "first"  %}
+            <script>
+                $('select').val( "{{ grading_url }}?view={{ view_all ? 'all' : '' }}&sort=first" )
+            </script>
+        {% elseif sort == "last"  %}
+            <script>
+                $('select').val( "{{ grading_url }}?view={{ view_all ? 'all' : '' }}&sort=last" )
+            </script>
+        {% endif %}
```

Comment:
A couple of suggestions here:
1. We can use the shorthand ternary statement in twig to condense a few bits, so that `{{ cond ? 'foo' }}` is equivalent to `{{ cond ? 'foo' : '' }}`.
2. Let's inline our selection into the HTML, instead of having this separate JS, by doing `{{ sort == "foo" ? 'selected' }}`
3. Since the start of the URL for each option is the same (`{{ grading_url }}?view={{ view_all ? 'all' }}`), let's move this into the `onChange` handler to reduce having it in every option.
```suggestion
        <select
            id="subsection_sort"
            name="subsection_sorting"
            onChange="window.location.href='{{ grading_url }}&view={{ view_all ? 'all' }}&sort=' + this.value"
            style="font-size:15px;"
        >
            <option value="section_subsection" aria-label="sort by Section then Subsection" {{ sort == "section_subsection" ? 'selected' }}>
                Section then Subsection then User ID
            </option>
            <option value="id" aria-label="sort by ID" {{ sort == "id" ? 'selected' }}>
                Section then User ID
            </option>
            <option value="first" aria-label="sort by First Name" {{ sort == "first" ? 'selected' }}>
                Section then First Name then User ID
            </option>
            <option value="last" aria-label="sort by Last Name" {{ sort == "last" ? 'selected' }}>
                Section then Last Name then User ID
            </option>
        </select>
```

You'll want to double check the `onChange` clause as I freehand wrote that so it might not be perfectly right.